cron/k8s/cron.yaml

```diff
@@ -32,7 +32,7 @@ spec:
               valueFrom:
                 secretKeyRef:
                   name: github
-                  key: token
+                  key: test_token
```

Comment:
can even start deleting all old cron code, after first iteration of pubsub works.

Reply:
Agreed.